pkg/meta/meta.go

```diff
@@ -174,8 +173,7 @@ func processCreate(ctx context.Context, astmt spqrparser.Statement, mngr EntityM
 		return cli.AddShardingRule(ctx, shardingRule)
 	case *spqrparser.KeyRangeDefinition:
 		req := kr.KeyRangeFromSQL(stmt)
-		if err := ops.AddKeyRangeWithChecks(ctx, mngr.QDB(), req); err != nil {
```

Comment:
Why do so?

Reply:
Because AddKeyRangeWithChecks doesn't work with adapter and after we are calling this method router shutting down with error.